core/domain/rights_manager.py

```diff
@@ -731,17 +731,21 @@ def check_can_edit_activity(user, activity_rights):
     if role_services.ACTION_EDIT_OWNED_ACTIVITY not in user.actions:
         return False
 
-    if (activity_rights.is_owner(user.user_id) or
+    if (
+            activity_rights.is_owner(user.user_id) or
             activity_rights.is_editor(user.user_id)):
         return True
 
-    if (activity_rights.community_owned or
+    if (
+            activity_rights.community_owned or
             (role_services.ACTION_EDIT_ANY_ACTIVITY in user.actions)):
         return True
 
-    if (activity_rights.is_published() and
-            (role_services.ACTION_EDIT_ANY_PUBLIC_ACTIVITY in
-             user.actions)):
+    if (
+            activity_rights.is_published() and
+            (
```

Comment:
Can we move this bracket to previous line? (May be try out putting 745 and 746 to line 744?

Reply:
Done moved to line 744 and 745